packages/types/tsconfig.json

```diff
@@ -14,5 +14,6 @@
         "rootDir": "./src",
         "composite": true
     },
-    "exclude": ["**/node_modules", "**/lib", "**/*.test.js", "**/*.test.ts"]
+    "exclude": ["**/node_modules", "**/lib", "**/*.test.js", "**/*.test.ts"],
+    "typeRoots": ["node_modules/@types"],
```

Comment:
What is this needed for?

Reply:
good question ... thats none of my changes ... might be a broken merge